src/index.ts

```diff
@@ -1,14 +1,40 @@
-import express from "express";
-const app = express();
-const port = 8080; // default port to listen
+import { fetchOrderbook } from "./fetch_orderbook"
+import {
+  Orderbook,
+  transitiveOrderbook
+} from "@gnosis.pm/dex-contracts/src/orderbook"
+import express from "express"
 
-// define a route handler for the default home page
-app.get("/", (req, res) => {
-  res.send("Hello world!");
-});
+const app = express()
+const port = 8080 // default port to listen
+
+// Setup orderbook fetching every 1000s
+let orderbooks: Map<string, Orderbook>
+setInterval(async () => {
+  orderbooks = await fetchOrderbook()
+}, 1000)
```

Comment:
nit: I would move this to the orderbook module, and then have a `getCurrentOrderbook` method for getting the currently cached orderbook. It looks like the current orderbook is getting cached in two places right now.

Reply:
I refactored the orderbook fetching method to be a class which exposes the latest orderbook as a value.